Yoast/wordpress-seo
# Introduce new Media tab

## Summary

This PR can be summarized in the following changelog entry:

* Introduce a new setting to redirect attachment URLs. Previously, we had an option to redirect attachment URLs to their post parent. This didn't work for attachments that weren't attached to anything. This new setting redirects *all* attachment URLs to the URL of the original image / media item. This is all explained on the new Media tab under Titles & Meta's.

## Relevant technical choices:

* Created a new tab Media.
* Changed the attachment redirect function.
* Added some helper function in `class-upgrade.php`. 
* Performed some cleanup to get rid of globals on the tabs of Titles & Meta's.

## Test instructions

This PR can be tested by following these steps:

* Enable the feature, attachment URLs should redirect.
* Check that attachment URLs are not in the sitemap when the feature is on OR when attachment are set to not appear in search results.

## Quality assurance


Fixes #8797
